README.en.md

```diff
@@ -80,56 +84,81 @@ Example:
 	}
 }
 ```
-
-### 設定パラメータの意味
-* token : string
- * APIトークン（https://api.slack.com/docs/oauth-test-tokens で取得できる）
+### Parameters
+#### interfaces : any[] 
+* settings about devices
+* type: "stdio"
+	+ standard input & output. No parameters to set.
+* type: "slack"
+	+ team : string
+		- name of your team `TEAMNAME.slack.com`
+	+ token : string
+		- API Token (Available: https://api.slack.com/bot-users)
+		- After you get token, you should add bot in slack>Apps, and then invite the bot to teams.
```

Comment:
After you get a token, you should add a bot in slack>Apps, and then invite the bot to teams.